pepper-apis/src/main/java/org/broadinstitute/ddp/util/CollectionMiscUtil.java

```diff
@@ -65,4 +67,13 @@
     public static Collector<CharSequence, ?, String> joinWithComma() {
         return Collectors.joining(",");
     }
+
+    public static boolean anyStartsWithIgnoreCase(List<String> strList, String prefix) {
+        if (strList != null) {
+            strList
+                    .stream()
+                    .anyMatch(str -> StringUtils.startsWithIgnoreCase(str, prefix));
```

Comment:
Should we really ignore case? These are identifiers so I don't think it'll be a good idea...

Reply:
Yes, agree. Fixed.